### ISSUE TITLE
Terminal: NVDA does not read the shortcut command of Add Terminal and Split Terminal Properly.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52283]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: NVDA2018.2

**Repro Steps:**
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Integrated Terminal.
3. Tab till Add terminal button(+).

**Actual:**
NVDA does not read the shortcut command of Add Terminal(Ctrl+Shift+`) and split Terminal(Ctrl+\) properly. It only reads Ctrl+shift button and ctrl+ button. Same happens while navigating to Terminal(Ctrl+`).

**Expected:**
NVDA should read the whole command Ctrl Shift Backtick key and Ctrl +BackSlash key like that.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not get the command properly and will not be able to navigate to the particular section.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode Terminal NVDA Shortcut Commands.pptx](https://github.com/Microsoft/vscode/files/2114341/A11y.VSCode.Terminal.NVDA.Shortcut.Commands.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This feels like low hanging fruit - the actions should just have proper titles
Assigning to july, @Tyriar feel free to move out

**Comment by Tyriar** [comment]
@isidorn how are the labels exposed to the screen reader in the panel actions? I only see a `title` which screen readers ignore?

![image](https://user-images.githubusercontent.com/2193314/42338664-6d375aa2-803f-11e8-9922-1fe089ae0b67.png)

**Comment by isidorn** [comment]
@Tyriar the title should be enough and this works perfeltly fine with latest vscode insiders and nvda on windows which aligns with that.
@AccessibilityTestingTeam-TCS can you still reproduce this?
![screen shot 2018-07-06 at 12 34 55](https://user-images.githubusercontent.com/1926584/42374640-095ff996-8119-11e8-9709-9396830c467d.png)

**Comment by Tyriar** [comment]
@isidorn well in that case I'm not so sure we can fix this, we're giving "ctrl+shift+`" and "ctrl+\" to the screen reader and it's not reading it. I guess we could have a list of characters that screen readers don't read and keep translations for each of them, this should be at the keybindings system level though.

**Comment by Tyriar** [comment]
@alexandrudima any thoughts on localizing common keybinding character names to fix this?

**Comment by AccessibilityTestingTeam-TCS** [comment]
@isidorn shortcut for Split Terminal has been changed(Ctrl+shift+5) so this is fine but shortcut for Add Terminal is still same(Ctrl+Shift+') which screenreader is not reading. And found same issue with Zoom out(Ctrl+-).

**Comment by Tyriar** [comment]
Removing myself as this applies to keybinding strings in general.

**Comment by isidorn** [comment]
Ok so the issue is that the keybindings are not localized and nvda is not reading them. Removing milestone and leaving to @alexandrudima to decide here

**Comment by alexandrudima** [comment]
The text we place in the dom is correct. It just happens that NVDA reads the text as if it were prose, omitting to pronounce backticks (`) or commas (,) or dots (.), etc.

The same problem exists when editing source code. Source code is full with such symbols. I believe the solution is that NVDA has some mode where it can read things letter-by-letter.

=> Closing, as we do the correct thing and this is something end users can control from the screen reading software.
